solidity-v1/dashboard/package-lock.json

```diff
@@ -3216,7 +3216,7 @@
     "@stablelib/binary": {
       "version": "0.7.2",
       "resolved": "https://registry.npmjs.org/@stablelib/binary/-/binary-0.7.2.tgz",
-      "integrity": "sha1-GzOSFwyKh0HIuPhD6ilN5xrrLPc=",
+      "integrity": "sha512-J7iGppeKR112ICTZTAoALcT3yBpTrd2Z/F0wwiOUZPVPTDFTQFWHZZdYzfal9+mY1uMUPRSEnNmDuXRZbtE8Xg==",
```

Comment:
I can't see any changes in `package.json` file so I think `pacakge-lock.json` should not be updated 🤔.

Reply:
19ecb12dc0b9e1ecf836865841199a7a739b50ac